### PR TITLE
Run e2e tests on KinD

### LINKED ISSUE #3: Add working e2e tests
**Describe the bug**
In order to run e2e tests for camel, we need to install some software looks like. We should add them, and also add an automagic release.

https://github.com/knative/test-infra/blob/master/guides/release_setup.md#setting-up-automated-releases

 **Expected behavior**

 **To Reproduce**
Launch the controller, it fails to start with:
E0915 22:33:42.994066       1 reflector.go:153] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:105: Failed to list *v1.Integration: the server could not find the requested resource (get integrations.camel.apache.org)

 **Knative release version**

 **Additional context**
Add any other context about the problem here such as proposed priority

### PR DESCRIPTION
**Pull request by nicolaferraro**
Fixes #3 

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e/camel_source_test.go [hunk 1]
```diff
@@ -19,11 +19,13 @@ limitations under the License.
 package e2e
 
 import (
+	camelv1 "github.com/apache/camel-k/pkg/apis/camel/v1"
+	camelclientset "github.com/apache/camel-k/pkg/client/camel/clientset/versioned"
```
Comment (mattmoor) [inline test/e2e/camel_source_test.go:23]: Format Go code:
```suggestion

```

File: test/e2e/camel_source_test.go [hunk 2]
```diff
@@ -19,11 +19,13 @@ limitations under the License.
...
+	"os"
+	"strings"
 	"testing"
 	"time"
```
Comment (mattmoor) [inline test/e2e/camel_source_test.go:27]: Format Go code:
```suggestion
	"time"

	camelv1 "github.com/apache/camel-k/pkg/apis/camel/v1"
	camelclientset "github.com/apache/camel-k/pkg/client/camel/clientset/versioned"
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**